### PR TITLE
Add section on curation to venue index page.

### PR DESCRIPTION
**Pull request by Daniel-Mietchen**
A first attempt at 
- making the curation pages more prominent, as per #1122 
- exploring how to integrate with TABernacle, as per #516 
- exploring the need for a generic missing page for an aspect, e.g. as per #720 

Not sure this is the way to go, so see this as a basis for discussion.

### CODE REVIEW

**Comment by Daniel-Mietchen** [comment]
Attaching screenshot.
Screenshot_2020-05-10 Scholia

**Comment by egonw** [comment]
What about introducing the three-column design we already have on so many other pages?

File: scholia/app/templates/venue_empty.html [hunk 1]
```diff
@@ -41,6 +41,14 @@ <h3>Multiple venues</h3>
 </ul>
 
 
-{% endblock %}
+<h3>Curation</h3>
+
+You can help improve Scholia profiles by curating the underlying data in Wikidata.
+
+For instance, the profile of each individual venue links to a dedicated curation page like <a href="../venue/Q15749954/missing">this one</a> for the <a href="/venue/Q15749954">Malaria Journal</a>, which is <a href="https://www.wikidata.org/entity/Q15749954">Q15749954</a> on Wikidata.
```
Comment (fnielsen) [inline scholia/app/templates/venue_empty.html:48]: `href="/venue/Q15749954"` will not work on tools.wmflabs.org

Comment (fnielsen) [inline scholia/app/templates/venue_empty.html:48]: Here is another one with the missin "../

File: scholia/app/templates/venue_empty.html [hunk 2]
```diff
@@ -41,6 +41,14 @@ <h3>Multiple venues</h3>
...
 
+Another way to assist with curation would be to use the TABernacle tool to <a href="https://tools.wmflabs.org/tabernacle/?#/tab/sparql/SELECT%20DISTINCT%20%3Fitem%20%3FitemLabel%20%0AWHERE%20%7B%0A%20%20hint%3AQuery%20hint%3Aoptimizer%20%22None%22.%0A%20%20SERVICE%20wikibase%3Amwapi%20%7B%0A%20%20%20%20bd%3AserviceParam%20wikibase%3Aapi%20%22Search%22%3B%0A%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20wikibase%3Aendpoint%20%22www.wikidata.org%22%3B%0A%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20mwapi%3Asrsearch%20%22%20%20haswbstatement%3AP236%22%20.%20%0A%20%20%20%20%3Ftitle%20wikibase%3AapiOutput%20mwapi%3Atitle.%0A%20%20%7D%0A%20%20BIND(IRI(CONCAT(STR(wd%3A)%2C%20%3Ftitle))%20AS%20%3Fitem)%0A%20%20FILTER%20NOT%20EXISTS%20%7B%20%3Fitem%20wdt%3AP921%20%5B%5D%20%7D%0A%20%20SERVICE%20wikibase%3Alabel%20%7B%20bd%3AserviceParam%20wikibase%3Alanguage%20%22%5BAUTO_LANGUAGE%5D%2Cen%22.%20%7D%0A%7D%0ALIMIT%201000/Len%3BDen%3BAen%3BP407%3BP921">add "main subject" statements to items about venues</a> for which Wikidata is missing that information.
```
Comment (fnielsen) [inline scholia/app/templates/venue_empty.html:51]: What are the new URL pattern? Are we suppose to use https://tabernacle.toolforge.org?

Comment (fnielsen) [reply scholia/app/templates/venue_empty.html:51]: No webservice there.

File: scholia/app/templates/venue_empty.html (hunk 1 above)
Comment (fnielsen) [reply scholia/app/templates/venue_empty.html:48]: The URL could just be "Q15749954" I think.

**Comment by egonw** [comment]
`tox -e flake8` passed.

**Comment by Daniel-Mietchen** [comment]
I had another look at the changes you suggested. Made some edits, but now tox says no &mdash; not sure yet why.

**Comment by Daniel-Mietchen** [comment]
I'm closing this one to restart afresh.

**Pull request closed without merging**